src/value.c

```diff
@@ -30,65 +32,55 @@ SIValue SI_DoubleVal(double d) {
   return (SIValue){.doubleval = d, .type = T_DOUBLE};
 }
 
-SIValue SI_StringVal(const char *s) {
-  return (SIValue){.stringval = strdup(s), .type = T_STRING};
+inline SIValue SI_NullVal() {
```

Comment:
Why inline this function?

Reply:
Honestly, I doubt that `inline` has any effect here! I didn't modify this function; I just moved it to match the declaration order in the header and standardized its layout. Agreed, though, let me drop that.